CMAQ.config.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
We will need to support docker/nci configurgation files. 
Maybe this should be moved to a path like config/cmaq/config.docker.json